service/src/main/kotlin/no/nav/su/se/bakover/service/søknadsbehandling/SøknadsbehandlingServiceImpl.kt

```diff
@@ -579,46 +573,15 @@ internal class SøknadsbehandlingServiceImpl(
             return it.left()
         }
 
-        return vilkårsvurder(
-            SøknadsbehandlingService.VilkårsvurderRequest(
-                behandlingId = søknadsbehandling.id,
-                behandlingsinformasjon = søknadsbehandling.behandlingsinformasjon.oppdaterBosituasjonOgEktefelleOgNullstillFormueForEpsHvisIngenEps(
-                    bosituasjon = bosituasjon,
-                ).getOrHandle { return KunneIkkeLeggeTilBosituasjonEpsGrunnlag.KlarteIkkeHentePersonIPdl.left() },
-            ),
-        ).mapLeft {
-            return KunneIkkeLeggeTilBosituasjonEpsGrunnlag.FantIkkeBehandling.left()
+        return søknadsbehandling.oppdaterBosituasjon(bosituasjon, clock) {
+            personService.hentPerson(it)
```

Comment:
Det skal gå an å kalle suksessfullt på `override fun leggTilBosituasjonEpsgrunnlag` for en saksbehandler som ikke har tilgang til denne personen (kode 6 etc). Noe denne snutten var ment å løse:
```
personService.hentPerson(it).fold(
                { error ->
                    if (error is KunneIkkeHentePerson.IkkeTilgangTilPerson) {
                        true.right()
                    } else {
                        KunneIkkeLeggeTilBosituasjonEpsGrunnlag.KlarteIkkeHentePersonIPdl.left()
                    }
                },
                {
                    true.right()
                },
            )
```
Det ser ikke ut som denne hentPerson-parameteren i Søknadbehandling brukes lenger uansett? Så den kan nok fjernes.

Reply:
mener du at tilgangs sjekken som blir gjort i request.toBosituasjon() ikke lenger er nødvendig?